test/postgresql/import_test.rb

```diff
@@ -1,4 +1,7 @@
 require File.expand_path(File.dirname(__FILE__) + '/../test_helper')
 require File.expand_path(File.dirname(__FILE__) + '/../support/postgresql/import_examples')
 
+pg_version = ENV["PG_VERSION"] || "0"
```

Comment:
I had to look through the code a bit, but this [wiki entry](https://github.com/zdennis/activerecord-import/wiki/How-to-run-the-tests) really helped.